src/lerobot/policies/act/modeling_act.py

```diff
@@ -79,7 +81,7 @@ def __init__(
 
         self.reset()
 
-    def get_optim_params(self) -> dict:
+    def get_optim_params(self) -> object:
```

Comment:
Using object here feels too generic and makes the typing less helpful. A stricter approach would be:

Option A (inline type):

`Sequence[dict[str, list[nn.Parameter] | float]]`

Option B (preferred, clearer): define a dedicated `TypedDict` for optimizer kwargs and then use:
```python
from typing import TypedDict, NotRequired
from collections.abc import Sequence
from torch import nn

class OptimizerKwargs(TypedDict, total=False):
    params: list[nn.Parameter]
    lr: float
    weight_decay: float
    momentum: float
    betas: tuple[float, float]
    eps: float

BatchKwargs = Sequence[OptimizerKwargs]
```
Option B has the advantage of being more self-documenting and makes it easier to see what fields are expected in the optimizer kwargs.